### PR TITLE
Add option to ignore imports from the Python stdlib

### PR DESCRIPTION
**Pull request by Nicolas-Reyland**
I would like to add an option to ignore imports of the python standard library.

### CODE REVIEW

**Comment by Nicolas-Reyland** [comment]
I just wrote a 'prototype' version. I would just like to know if you would appreciate such an option. If this is not the case, I won't have to improve my code/write tests, etc.

**Comment by Nicolas-Reyland** [comment]
If you plan to do so, please do not merge this yet. There is still some work to be done.

File: findimports.py
Comment (mgedmin) [reply findimports.py:712]: (This probably explains the test failures!)

File: findimports.py
Comment (Nicolas-Reyland) [reply findimports.py:311]: Ok thank you, I'll be looking into it !

File: findimports.py
Comment (Nicolas-Reyland) [reply findimports.py]: I called `list` on this. Is it ok that way or do you prefer a list comprehension with an `if` clause ?

File: findimports.py
Comment (Nicolas-Reyland) [reply findimports.py]: I used the set given in python3.10 and used copy-pasted it (reformatted for flake8 etc).

**Comment by Nicolas-Reyland** [comment]
This sounds all good to me. Let me know if you want anything changed!

**Comment by Nicolas-Reyland** [comment]
This look all good to me. Feel free to review/change my code.

File: findimports.py
Comment (mgedmin) [inline findimports.py:178]: I'd sort this alphabetically, but I suppose it doesn't really matter.

As for my earlier mention of the Python 3.10 feature, I wasn't suggesting bumping the minimum required Python version, but maybe it's worth using it directly, if it exists, so that new stdlib modules appearing in future Python versions will automatically be recognized?  I.e. something like

```python
STDLIB_MODNAMES_SET = getattr(sys, 'stdlib_module_names', frozenset([
    # taken from Python 3.10
    '__future__', '_abc', ...
]))
```

File: findimports.py
Comment (mgedmin) [inline findimports.py:579]: I really dislike `filter(lambda ...)`.  If you don't want replace it with a list comprehension I'll do that myself after the merge.

File: tests/unknown-modules.txt
Comment (mgedmin) [inline tests/unknown-modules.txt:28]: Style tip: always use keyword-argument syntax when passing boolean literals, so one doesn't have to look up the method definition to figure out what False means here.

**Review by mgedmin — APPROVED** [review]
Mostly looks good to me.  The remaining suggestions are mostly stylistic that I can do myself.

Since I'm a very lazy person I'll wait a day or two to see if you're willing to do them, before hitting the merge button. ;)

Thank you!

File: findimports.py
Comment (Nicolas-Reyland) [reply findimports.py:579]: No worries, I didn't know exactly what you meant at the time. My bad

**Comment by Nicolas-Reyland** [comment]
I've resolved all the issues. Please let me know if there's anything else !

**Comment by Nicolas-Reyland** [comment]
Ok, I pushed to the wrong branch ....

**Review by mgedmin — APPROVED** [review]

**Comment by mgedmin** [comment]
Thank you!

**Comment by mgedmin** [comment]
Merged and released findimports 2.2.0

**Pull request merged**
